.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,57 @@
+name: Docker
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main", "master", "develop" ]
+    # Publish semver tags as releases.
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-*'
```

Comment:
Hmm, not sure what'll happen when we push a tag and also create a release on that tag. We may need to update this

Reply:
Why do you believe this could be a concern? Could you provide a more detailed explanation?

Reply:
We push tag 1.2.3 -> image build with tag 1.2.3
We make a release from that build -> image build with tag 1.2.3

Reply:
I assume they will be shown under `Recent tagged image versions`. Why is this a concern?

Reply:
I didn't say it was a concern, I said I'm not sure what'll happen and we should keep an eye on it